packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/TaskNavButtonsContainer.js

```diff
@@ -63,16 +59,11 @@ class TaskNavButtonsContainer extends React.Component {
       annotatedSteps,
       classification,
       selectStep,
-      step,
       tasks
     } = storeMapper(this.context)
     this.completeStepTasks()
-    const annotations = tasks.map(task => classification.annotation(task))
-    annotatedSteps.next({
-      step,
-      annotations
-    })
-    selectStep()
+    const { next } = annotatedSteps.latest.step
+    selectStep(next)
```

Comment:
I think it would be good to have the Next button update history here, so that both Back and Next buttons are responsible for updating history. Something like
```js
annotatedSteps.next({
  step: nextStep
  classification
})
```
where `nextStep` is the next step from the workflow's steps.